plan/physical.go

```diff
@@ -127,6 +127,9 @@ func WithDefaultMemoryLimit(memBytes int64) PhysicalOption {
 func OnlyPhysicalRules(rules ...Rule) PhysicalOption {
 	return physicalOption(func(pp *physicalPlanner) {
 		pp.clearRules()
+		// Always add physicalConverterRule. It doesn't change the plan but only convert nodes to physical.
+		// This is required for some pieces to work on the physical plan (e.g. SetTriggerSpec).
+		pp.addRules(physicalConverterRule{})
```

Comment:
I am sorry, I haven't seen that those changes remained the same from my previous push.
I added them because of how i managed tests before. @wolffcm do you want me to keep them?

Reply:
This one seems pretty logical, because `physicalConverterRule` can't be added manually once removed.